etc/scripts/licenses/buildrules.py

```diff
@@ -222,6 +222,8 @@ def cli(licenses_file):
             base_name = "false-positive"
         elif rule.is_license_intro:
             base_name = "license-intro"
+        elif rule.is_license_clue:
+            base_name = "license-clue"
```

Comment:
Should not we use `f"license-clue_{rule.icense_expression}"` instead here so the name gives a hint of what the clue is about?

Reply:
Yeah, this makes more sense. Adding this.